lib/node_modules/@stdlib/stats/base/dists/boltzmann/pmf/README.md

```diff
@@ -0,0 +1,159 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# Probability Mass Function
+
+> [Boltzmann][boltzmann-distribution] distribution probability mass function (PMF).
+
+<section class="intro">
+
+The [probability mass function][pmf] (PMF) for a [boltzmann][boltzmann-distribution] random variable is
+
+```math
+f(x; \lambda, N) = P(X=x; \lambda, N) = \begin{cases}
+\frac{{(\exp(-\lambda k))}{(1-\exp(-\lambda))}}{{(1-\exp(-\lambda N))}} & \text{for } k = 0, 1, 2, \ldots, N \\0 & \text{otherwise}
+\end{cases}
+```
```

Comment:
This isn't how we insert equations. You only need to include the HTML comment and then we have build tooling which auto-inserts.